src/transformers/modeling_utils.py

```diff
@@ -808,7 +809,13 @@ def _load_state_dict_into_meta_model(
     for old_key, new_key in zip(old_keys, new_keys):
         state_dict[new_key] = state_dict.pop(old_key)
 
-    for param_name, param in state_dict.items():
+    # Show shard-level progress. Useful to monitor quantization progress
+    show_progress = False
```

Comment:
```suggestion
    quant_show_progress = False
```
As this is specific to quantization